### PR TITLE
🛠 #1254 Small bug fixes

### PR DESCRIPTION
**Pull request by stephanevieira75**
### Remove Change company title in user companies list when there is only one company
Capture d’écran de 2021-06-22 15-38-22

### Update autocomplete placeholder in ChannelMembersList
Capture d’écran de 2021-06-22 15-40-54

### Rename 'it's you' tag by 'You'
Capture d’écran de 2021-06-22 15-41-15

### Improve antd checkbox border color

https://user-images.githubusercontent.com/36481167/122935497-abf9ba00-d370-11eb-91aa-4d2978ede00b.mp4

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/WorkspacesBar/Components/Group.js [hunk 1]
```diff
@@ -88,7 +92,8 @@ export default class Group extends Component {
         text: Languages.t('scenes.app.workspacesbar.components.create_company_menu'),
         icon: 'plus',
         onClick: () => {
-          popupManager.open(<CreateCompanyView />);
+          //popupManager.open(<CreateCompanyView />);
+          console.log(Groups.user_groups);
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/WorkspacesBar/Components/Group.js:96]: Is it expected this console.log + the fact that the button does nothing now?

Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/WorkspacesBar/Components/Group.js]: Definitly not excepted, back to previous state. Thank you :+1:

**Comment by RomaricMourgues** [comment]
@hkotthapalli @BenoitTallandier could you check this new translations? Thanks! 
https://github.com/linagora/Twake/pull/1369/commits/ff7465b14105cee002d9eb2c8da96f1a77bf9973

**Review by chamerling — APPROVED** [review]

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 2]
```diff
@@ -1829,7 +1829,7 @@ export default {
   'scenes.apps.calendar.modals.reminder_add': 'Ajouter un rappel',
   'scenes.apps.drive.drive_tour.left_bar.not_in_ws':
     "L'utilisateur n'est pas dans l'espace de travail courant",
-  'scenes.apps.drive.drive_tour.left_bar.imported': 'Importé depuis un autre espace de travail',
+  'scenes.apps.drive.drive_tour.left_bar.imported': 'Téléchargé depuis un autre espace de travail',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:1832]: Ici c'est bien "importé"

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 3]
```diff
@@ -1270,7 +1270,7 @@ export default {
   'scenes.app.channelsbar.channel_leaving': 'Quitter la chaîne',
   'scenes.app.channelsbar.channel_removing': 'Supprimer la chaîne',
   'scenes.app.channelsbar.company_invitation_alert_subtitle':
-    'Vous pouvez importer ces chaînes dans votre entreprise pour rester organisé en cliquant sur ce cadre.',
+    'Vous pouvez Télécharger ces chaînes dans votre entreprise pour rester organisé en cliquant sur ce cadre.',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:1273]: Ici c'est bien "importé"

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 4]
```diff
@@ -1131,7 +1131,7 @@ export default {
   'scenes.app.workspaces.create_company.group_data.title':
     'Faisons vite, nous avons juste besoin de quelques informations statistiques',
   'scenes.app.workspaces.create_company.importations.title_1':
-    'Vous travaillez déjà avec des outils numériques ? Importez ou intégrez vos outils dès maintenant !',
+    'Vous travaillez déjà avec des outils numériques ? Téléchargez ou intégrez vos outils dès maintenant !',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:1134]: Ici c'est bien "importez"

File: twake/frontend/src/app/services/languages/locale/fr.js (hunk 3 above)
Comment (RomaricMourgues) [reply twake/frontend/src/app/services/languages/locale/fr.js:1273]: *importer

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 5]
```diff
@@ -355,8 +355,8 @@ export default {
   'scenes.apps.calendar.calendar_modal.delete_message':
     'Supprimer le calendrier et tous les événements de celui-ci.',
   'scenes.apps.calendar.calendar_modal.participants': 'Participants',
-  'scenes.apps.calendar.calendar_modal.importation': 'Importation',
-  'scenes.apps.calendar.calendar_modal.importation_message': 'Importer un fichier ics',
+  'scenes.apps.calendar.calendar_modal.importation': 'Télécharger',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:358]: C'est bien "Importation"

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 6]
```diff
@@ -1756,7 +1756,7 @@ export default {
   'scenes.app.mainview.create_account': 'Create your workspace for free on ',
   'scenes.apps.calendar.modals.reminder_add': 'Add a reminder',
   'scenes.apps.drive.drive_tour.left_bar.not_in_ws': 'User not in current workspace',
-  'scenes.apps.drive.drive_tour.left_bar.imported': 'Imported from another workspace',
+  'scenes.apps.drive.drive_tour.left_bar.imported': 'Uploaded from another workspace',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:1759]: It is still "Imported"

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 7]
```diff
@@ -1212,7 +1212,7 @@ export default {
   'scenes.app.channelsbar.channel_leaving': 'Leave channel',
   'scenes.app.channelsbar.channel_removing': 'Delete channel',
   'scenes.app.channelsbar.company_invitation_alert_subtitle':
-    'You can import these channels into your business to stay organized by clicking on this frame.',
+    'You can upload these channels into your business to stay organized by clicking on this frame.',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:1215]: It is still "import"

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 8]
```diff
@@ -1078,7 +1078,7 @@ export default {
   'scenes.app.workspaces.create_company.group_data.title':
     " We'll be quick — we just need some statistical information.",
   'scenes.app.workspaces.create_company.importations.title_1':
-    'Already working with digital tools? Import or integrate your tools now!',
+    'Already working with digital tools? Upload or integrate your tools now!',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:1081]: It is still "Import"

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 9]
```diff
@@ -343,8 +343,8 @@ export default {
   'scenes.apps.calendar.calendar_modal.shared_with': 'Shared with',
   'scenes.apps.calendar.calendar_modal.delete_message': 'Delete calendar and all events in it.',
   'scenes.apps.calendar.calendar_modal.participants': 'Participants',
-  'scenes.apps.calendar.calendar_modal.importation': 'Import',
-  'scenes.apps.calendar.calendar_modal.importation_message': 'Import an ics file',
+  'scenes.apps.calendar.calendar_modal.importation': 'Upload',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:346]: Here it is "Import" (but you can keep Upload for 'Upload an ics file',)

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 10]
```diff
@@ -80,14 +80,14 @@ export default {
...
-  'scenes.app.workspaces.create_company.group_data.group_size.1000_5000': '1000 - 4999 people',
-  'scenes.app.workspaces.create_company.group_data.group_size.5000_more': 'More than 5000 people',
+  'scenes.app.workspaces.create_company.group_data.group_size.less_3': 'Less than 4 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.4_10': '4 - 10 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.10_50': '11 - 49 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.50_100': '50 - 99 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.100_500': '100 - 499 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.500_1000': '500 - 999 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.1000_5000': '1000 - 4999 users',
+  'scenes.app.workspaces.create_company.group_data.group_size.5000_more': 'More than 5000 users',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:90]: @BenoitTallandier for this ones maybe not great?

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 11]
```diff
@@ -1078,13 +1078,13 @@ export default {
...
-    'Already working with digital tools? Upload or integrate your tools now!',
+    'Already working with digital tools? Import or integrate your tools now!',
   'scenes.app.workspaces.create_company.importations.title_2': 'No worries, you can do this later!',
   'scenes.app.workspaces.create_company.invitations.auto_add_inputs_info':
     'The number of fields increases when you press enter.',
   'scenes.app.workspaces.create_company.invitations.title_1':
-    'Before you finish, you can add people to your company.',
-  'scenes.app.workspaces.create_company.invitations.title_2': 'Add your team people.',
+    'Before you finish, you can add users to your company.',
+  'scenes.app.workspaces.create_company.invitations.title_2': 'Add your team users.',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/en.js:1087]: @BenoitTallandier for this one maybe not great?

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 12]
```diff
@@ -85,14 +85,16 @@ export default {
...
+  'scenes.app.workspaces.create_company.group_data.group_size.less_3': 'Moins de 4 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.4_10': '4 - 10 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.10_50': '11 - 49 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.50_100': '50 - 99 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.100_500': '100 - 499 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.500_1000': '500 - 999 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.1000_5000':
+    '1000 - 4999 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.5000_more':
+    'Plus de 5000 utilisateurs',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:97]: @BenoitTallandier for this ones maybe not great?

File: twake/frontend/src/app/services/languages/locale/fr.js [hunk 13]
```diff
@@ -36,14 +36,14 @@ export default {
...
-    "Besoin de plus de fonctionnalités ? Passez en version complète pour pouvoir ajouter plus de personnes, plus d'espaces de travail, espace de stockage illimité, etc.",
+    "Besoin de plus de fonctionnalités ? Passez en version complète pour pouvoir ajouter plus de utilisateurs, plus d'espaces de travail, espace de stockage illimité, etc.",
   'ads.activate_premium_to_add_workspaces':
     "Passez en version complète pour ajouter d'autres espaces de travail.",
   'ads.two_month_offer': 'Essayez Twake Premium gratuitement sans conditions pendant deux mois.',
   'ads.two_month_offer_confirmation':
     'Utiliser la version complète de Twake avec ce groupe gratuitement pendant deux mois.',
   'tutorial.start_to_add_workspaces':
-    'Commencez par ajouter un espace de travail partagé ou demander à une personne de vous inviter dans un espace existant.',
+    'Commencez par ajouter un espace de travail partagé ou demander à un utilisateur de vous inviter dans un espace existant.',
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/services/languages/locale/fr.js:46]: @BenoitTallandier for this one maybe not great?

**Review by RomaricMourgues — COMMENTED** [review]
For you @BenoitTallandier

File: twake/frontend/src/app/services/languages/locale/fr.js (hunk 13 above)
Comment (BenoitTallandier) [reply twake/frontend/src/app/services/languages/locale/fr.js:46]: Oui, on peut mettre un truc " ou demander à quelqu'un" :)

File: twake/frontend/src/app/services/languages/locale/fr.js (hunk 12 above)
Comment (BenoitTallandier) [reply twake/frontend/src/app/services/languages/locale/fr.js:97]: This one is ok, it's by users so I would keep "utilisateur" :)

File: twake/frontend/src/app/services/languages/locale/en.js (hunk 11 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/services/languages/locale/en.js:1087]: `Add some users` ?

File: twake/frontend/src/app/services/languages/locale/en.js [hunk 14]
```diff
@@ -34,9 +34,9 @@ export default {
   twake_not_on_device_mobile: 'You can download our mobile app on the Android and Apple stores.',
   twake_not_on_device_browser: 'You must use an updated browser or download Twake desktop.',
   'tutorial.start_to_add_workspaces':
-    'Start to add a shared workspace or ask people to invite you in a shared workspace.',
+    'Start to add a shared workspace or ask users to invite you in a shared workspace.',
```
[the remark this answers is not in the document]
Comment (stephanevieira75) [reply twake/frontend/src/app/services/languages/locale/en.js:37]: `... or ask someone to invite you ....`?

File: twake/frontend/src/app/services/languages/locale/fr.js (hunk 13 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/services/languages/locale/fr.js:46]: Updated 👍🏻

File: twake/frontend/src/app/services/languages/locale/en.js (hunk 11 above)
Comment (RomaricMourgues) [reply twake/frontend/src/app/services/languages/locale/en.js:1087]: Add your team users. => Add users to your team.

File: twake/frontend/src/app/services/languages/locale/en.js (hunk 14 above)
[the remark this answers is not in the document]
Comment (RomaricMourgues) [reply twake/frontend/src/app/services/languages/locale/en.js]: Yes I prefer tyour version @stephanevieira75

[the remark this answers is not in the document]
Comment (stephanevieira75) [reply twake/frontend/src/app/services/languages/locale/en.js]: done 👍🏻

File: twake/frontend/src/app/services/languages/locale/en.js (hunk 11 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/services/languages/locale/en.js:1087]: Done 👍🏻

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**